test/demuxer_test.exs

```diff
@@ -110,8 +93,6 @@ defmodule Membrane.Matroska.DemuxerTest do
       ]
       |> Testing.Pipeline.start_link()
 
-    assert_pipeline_playback_changed(pipeline, _, :playing)
```

Comment:
don't you need to somehow "synchronize" with the playback of the pipeline (`assert_pipeline_play`)?